packages/website/src/index.html

```diff
@@ -56,8 +56,7 @@
     <!--placeholder1-->
     <div id="root"></div>
     <!--/placeholder1-->
-    <!-- FIXME we should use async instead of defer -->
-    <script src="/main.tsx" defer type="module"></script>
+    <script src="/main.tsx" async type="module"></script>
```

Comment:
Sorry, I meant this:
```suggestion
    <!-- FIXME we should use async instead of defer -->
    <script src="/src/main.tsx" defer type="module"></script>
```